packages/core/crates/core-hopr/src/adaptors/ping.rs

```diff
@@ -76,7 +79,17 @@ pub mod wasm {
         pub async fn ping(&self, peer: js_sys::JsString) {
             let x: String = peer.into();
             if let Some(converted) = core_network::PeerId::from_str(&x).ok() {
-                self.ping.write().await.ping(vec![converted]).await;
+                let mut pinger = self.ping.write().await;
+
+                let timeout = sleep(std::time::Duration::from_millis(30_000)).fuse();
```

Comment:
Should we make this configurable?

Reply:
It should. But the configuration object from cli args/config file is not directly mappable to the lower level object. It requires unifying the config passdown from the object I created... by removing the TypeScript intermediary object. This was waiting for most of the core being migrated, which is now. I can do that in another sweep to unify core configs and consolidate them under the hoprd config.